Packs/SplunkPy/Integrations/SplunkPyV2/SplunkPyV2.py

```diff
@@ -0,0 +1,4180 @@
+FETCH_LIMIT = max(min(200, FETCH_LIMIT), 1)
+MIRROR_LIMIT = 1000
+SPLUNK_INDEXING_TIME = 60
+PROBLEMATIC_CHARACTERS = [".", "(", ")", "[", "]"]
+REPLACE_WITH = "_"
+REPLACE_FLAG = params.get("replaceKeys", False)
+FIRST_FETCH_TIME = params.get("first_fetch", "10 minutes")
+PROXIES = handle_proxy()
+DEFAULT_DISPOSITIONS = {
+    "Unassigned": "disposition:0",
+    "True Positive - Suspicious Activity": "disposition:1",
+    "Benign Positive - Suspicious But Expected": "disposition:2",
+    "False Positive - Incorrect Analytic Logic": "disposition:3",
+    "False Positive - Inaccurate Data": "disposition:4",
+    "Other": "disposition:5",
+    "Undetermined": "disposition:6",
+}
+
+DEFAULT_STATUSES = {
+    "Unassigned": "0",
+    "Assigned": "1",
+    "In Progress": "2",
+    "Pending": "3",
+    "Resolved": "4",
+    "Closed": "5",
+}
+
+# =========== Mirroring Mechanism Globals ===========
+MIRROR_DIRECTION = {"None": None, "Incoming": "In", "Outgoing": "Out", "Incoming And Outgoing": "Both"}
+OUTGOING_MIRRORED_FIELDS = ["note", "status", "owner", "urgency", "reviewer", "disposition"]
+
+# === Note Tag Globals ===
+NOTE_TAG_TO_SPLUNK = params.get("note_tag_to_splunk", "FROM XSOAR")
+NOTE_TAG_FROM_SPLUNK = params.get("note_tag_from_splunk", "FROM SPLUNK")
+
+# =========== Enrichment Mechanism Globals ===========
+ENABLED_ENRICHMENTS = params.get("enabled_enrichments", [])
+
+DRILLDOWN_ENRICHMENT = "Drilldown"
+ASSET_ENRICHMENT = "Asset"
+IDENTITY_ENRICHMENT = "Identity"
+SUBMITTED_FINDINGS = "submitted_findings"
+EVENT_ID = "event_id"
+RULE_ID = "rule_id"
+ISO_FORMAT_TZ_AWARE = "%Y-%m-%dT%H:%M:%S.%f%z"  # e.g '2025-12-03T11:53:45.138540+00:00
+NOT_YET_SUBMITTED_FINDINGS = "not_yet_submitted_findings"
+INFO_MIN_TIME = "info_min_time"
+INFO_MAX_TIME = "info_max_time"
+INCIDENTS = "incidents"
+MIRRORED_ENRICHING_FINDINGS = "MIRRORED_ENRICHING_FINDINGS"
+PROCESSED_MIRRORED_EVENTS = "processed_mirror_in_events_cache"
+DUMMY = "dummy"
+ENRICHMENTS = "enrichments"
+MAX_HANDLE_FINDINGS = 20
+MAX_SUBMIT_FINDINGS = 30
+CACHE = "cache"
+STATUS = "status"
+DATA = "data"
+TYPE = "type"
+ID = "id"
+CREATION_TIME = "creation_time"
+QUERY_NAME = "query_name"
+QUERY_SEARCH = "query_search"
+INCIDENT_CREATED = "incident_created"
+
+DRILLDOWN_REGEX = r'([^\s\$]+)\s*=\s*"?(\$[^\s\$\\]+\$)"?|"?(\$[^\s\$\\]+\$)"?'
+
+ENRICHMENT_TYPE_TO_ENRICHMENT_STATUS = {
+    DRILLDOWN_ENRICHMENT: "successful_drilldown_enrichment",
+    ASSET_ENRICHMENT: "successful_asset_enrichment",
+    IDENTITY_ENRICHMENT: "successful_identity_enrichment",
+}
+COMMENT_MIRRORED_FROM_XSOAR = "***Mirrored from Cortex XSOAR***"
+USER_RELATED_FIELDS = ["user", "src_user"]
+
+# =========== Not Missing Events Mechanism Globals ===========
+CUSTOM_ID = "custom_id"
+OCCURRED = "occurred"
+INDEX_TIME = "index_time"
+TIME_IS_MISSING = "time_is_missing"
+
+
+# =========== Enrich User Mechanism ============
+class UserMappingObject:
+    def __init__(
```

Comment:
Can you add a docstring detailing the init args?

Reply:
done